src/Orchard.Specs/Blogs.feature

```diff
@@ -171,6 +171,7 @@ Scenario: I can create browse blog posts on several pages
         And I go to "admin/blogs"
         And I follow "My Blog"
         And I follow "New Post" where class name has "primaryAction"
+        And I wait "1000"
```

Comment:
These waits are needed, because the tests are ran using SQL CE and that doesn't have millisecond-precision, so the blogposts might be out of order if they are created in rapid succession, causing false negative test result at the end.